### PR TITLE
Read from admin dataset

### PR DESCRIPTION
**Pull request by mcarans**
This is a first draft at reading from the global p-codes dataset instead of from file. Needs tests etc. Also needs checking for performance issues.

### CODE REVIEW

**Comment by b-j-mills** [comment]
Do you think we should include any of the nesting information for admin levels > 1?  If in the future we're planning to match admin names to p-codes at admin 2, I've been imagining we'd make the match more sophisticated and try to match both parent and child names (thinking about duplicate/phonetically similar names).

Otherwise I think it's great!

**Comment by mcarans** [comment]
@b-j-mills Yes in future we could include nesting information

**Review by b-j-mills — APPROVED** [review]

**Comment by mcarans** [comment]
@turnerm This is ready for your review

File: src/hdx/location/adminlevel.py [hunk 1]
```diff
@@ -1,70 +1,162 @@
...
+    tried
 
     Args:
-        admin_config (Dict): Configuration dictionary
+        admin_config (Dict): Configuration dictionary. Defaults to {}.
         admin_level (int): Admin level. Defaults to 1.
         admin_level_overrides (Dict): Countries at other admin levels.
     """
 
+    _admin_url_int = "https://data.humdata.org/dataset/cb963915-d7d1-4ffa-90dc-31277e24406f/resource/f65bc260-4d8b-416f-ac07-f2433b4d5142/download/global_pcodes_adm_1_2.csv"
```
Comment (turnerm) [inline src/hdx/location/adminlevel.py:39]: I was initially a bit confused about the purpose of the two different `admin_url` variables. Not sure what you mean by "int" here but could you call this one `_admin_url_default` or `fallback` or something, that would make it more clear maybe?

**Review by turnerm — APPROVED** [review]
Looks a great from what I can tell! One tiny variable name comment :)

File: src/hdx/location/adminlevel.py (hunk 1 above)
Comment (mcarans) [reply src/hdx/location/adminlevel.py]: Good idea!

**Pull request merged**